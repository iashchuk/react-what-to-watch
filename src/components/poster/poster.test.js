import React from "react";
import renderer from "react-test-renderer";
import Poster from "./poster.jsx";

describe(`Poster`, () => {
  it(`renders correctly`, () => {
    const component = renderer.create(<Poster />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
