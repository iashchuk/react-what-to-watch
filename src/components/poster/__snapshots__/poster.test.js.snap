// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Poster renders correctly 1`] = `
<div
  className="movie-card__poster"
>
  <img
    alt="The Grand Budapest Hotel poster"
    height="327"
    width="218"
  />
</div>
`;
